test/c/basic/srem.c

```diff
@@ -0,0 +1,47 @@
+#include "smack.h"
+
+// @expect verified
+// @checkbpl grep ':= \$srem'
```

Comment:
Wait, isn't this always going to be true since there is always `$srem` in the prelude?
Am I missing something here?

Reply:
No, this pattern matches assignments that have `$srem` in the right-hand side.